control-plane/tests/test_phase52_6_3_legacy_import_alias_registry.py

```
@@ -0,0 +1,42 @@
+        self.assertIn(alias, registry.LEGACY_IMPORT_ALIASES)
+        self.assertEqual(registry.LEGACY_IMPORT_ALIASES[alias].target_module, owner)
+        self.assertEqual(registry.LEGACY_IMPORT_ALIASES[alias].target_family, "reporting")
+
+        legacy_module = importlib.import_module(alias)
+        owner_module = importlib.import_module(owner)
+        package = importlib.import_module("aegisops_control_plane")
+        from aegisops_control_plane import audit_export as legacy_from_package
+
+        self.assertIs(legacy_module, owner_module)
+        self.assertIs(legacy_from_package, owner_module)
+        self.assertIs(getattr(package, "audit_export"), owner_module)
```

Ruff (0.15.12)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)